internal/controller/validator.go

```diff
@@ -139,8 +141,45 @@ func (r *DataProtectionApplicationReconciler) ValidateDataProtectionCR(log logr.
 				appliedBackupSyncPeriod, appliedGarbageCollectionPeriod,
 			)
 		}
-		// TODO should also validate that BSL backupSyncPeriod is not greater or equal to nonAdmin.backupSyncPeriod
-		// but BSL can not exist yet when we validate the value
+
+		defaultBSLIndex := -1
+		bslsInOADPNamespace := &velerov1.BackupStorageLocationList{}
+		r.List(r.Context, bslsInOADPNamespace, client.InNamespace(r.NamespacedName.Namespace))
+		for index, bsl := range bslsInOADPNamespace.Items {
+			if bsl.Spec.Default {
+				defaultBSLIndex = index
+				break
+			}
```

Comment:
Is it safe to assume velero would be running by this time?

If not, we might want to either wait, or use the same logic as velero function, which would [remove older defaults](https://github.com/vmware-tanzu/velero/blob/a45c9f27e890380e889e073496132b53e89cb10c/pkg/controller/backup_storage_location_controller.go#L250-L255) if more than one.

Reply:
From my tests, If I just install OADP and create DPA with non admin enabled, NAC Pod goes live prior to Velero Pod, so we can not assume that :( 

Will take a look

Reply:
ah, but DPA reconcile will run every minute, so we do not need to wait, the next reconcile will have velero running, right?

Reply:
Sure. if image is correct.

Reply:
Why not add a check re: velero pod running ?

Reply:
I think this is separate problem

we could add validation to check if velero Pod is running, but I think this PR is not the place